docs/INSTALLATION.md

```diff
@@ -35,7 +35,17 @@ Update the `values.yaml` based on what you provisioned as part of the prerequisi
      * For more details, please refer to the **Volumes** section of the [configuration guide](CONFIGURATION.md).
      * You can also refer to our examples of AWS storage: [Local storage - utilizing AWS EBS-backed volumes](docs/examples/storage/aws/LOCAL_STORAGE.md) and [Shared storage - utilizing AWS EFS-backed filesystem](docs/examples/storage/aws/SHARED_STORAGE.md)    
     * Bitbucket needs a dedicated NFS server providing persistence for a shared home. Prior to installing the Helm chart, a suitable NFS shared storage solution must be provisioned. The exact details of this resource will be highly site-specific, but you can use this example as a guide: [Implementation of an NFS Server for Bitbucket](docs/examples/storage/nfs/NFS.md)
-
+ 
+ * **Bitbucket Secrets**: If you specify the names of the [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) that contain the Bitbucket license key and the Bitbucket sysadmin credentials, you will be able to bypass the license and the admin user configurations during the Bitbucket setup. To do this, update the `bitbucket` stanza in the `values.yaml` file:
+    * Specify `bitbucket.license.*` and `bitbucket.sysadminCredentials.*` like so:
```

Comment:
I think we can say the same thing as we do for the database, as there is already this in the `values.yaml` file:

```
  license:
    # -- The name of the Kubernetes Secret that contains the Bitbucket license key.
    # If specified, then the license will be automatically populated during Bitbucket setup.
    # Otherwise, it will need to be provided via the browser after initial startup.
    secretName:
    # -- The key in the Kubernetes Secret that contains the Bitbucket license key
    secretKey: license-key

  sysadminCredentials:
    # -- The name of the Kubernetes Secret that contains the Bitbucket sysadmin credentials
    # If specified, then these will be automatically populated during Bitbucket setup.
    # Otherwise, they will need to be provided via the browser after initial startup.
    secretName:
    # -- The key in the Kubernetes Secret that contains the sysadmin username
    usernameSecretKey: username
    # -- The key in the Kubernetes Secret that contains the sysadmin password
    passwordSecretKey: password
    # -- The key in the Kubernetes Secret that contains the sysadmin display name
    displayNameSecretKey: displayName
    # -- The key in the Kubernetes Secret that contains the sysadmin email address
    emailAddressSecretKey: emailAddress
```

so we can skip the example below.